.gitignore

```diff
@@ -6,3 +6,4 @@ build/
 dist/
 # Emacs backup files
 *~
+test/SBOLTestSuite
```

Comment:
Why is this being ignored? Is this part of the URL change? I don't think this is right.